packages/bsky/src/api/app/bsky/unspecced/getTimelineSkeleton.ts

```diff
@@ -48,7 +40,23 @@ export default function (server: Server, ctx: AppContext) {
             ref('originatorDid'),
           ]),
         )
+        .selectAll('feed_item')
+
+      const selfBuilder = ctx.db.db
+        .selectFrom('feed_item')
+        .where('feed_item.originatorDid', '=', viewer)
+        .selectAll('feed_item')
+
+      let feedItemsQb = ctx.db.db
+        .selectFrom(followBuilder.unionAll(selfBuilder).as('feed_item'))
+        .innerJoin('post', 'post.uri', 'feed_item.postUri')
         .where('feed_item.sortAt', '>', getFeedDateThreshold(sortFrom))
+        .selectAll('feed_item')
+        .select([
+          'post.replyRoot',
+          'post.replyParent',
+          'post.creator as postAuthorDid',
+        ])
```

Comment:
Just double-checking that this all works fine with pagination.  I only bring it up because I recall the way kysely places parentheses (or really, _doesn't_ place parentheses) with `union all` has given me some grief in the past.  I assume the tests would catch this, though.